buildSrc/src/main/groovy/VeracodeTask.groovy

```diff
@@ -68,15 +94,23 @@ abstract class VeracodeTask extends DefaultTask {
     }
 
     protected UploadAPIWrapper loginUpdate() {
```

Comment:
It is because they are wrapping two different APIs. In a future PR:
https://github.com/calgaryscientific/veracode-gradle-plugin/blob/eca51169f0b1c7b2b71f4f1a63e41392fe486333/buildSrc/src/main/groovy/VeracodeTask.groovy#L98
I'll rename this to uploadAPI() to make it clear. The loginResults() will be renamed to resultsAPI() as well.